src/tox.ini

```diff
@@ -180,8 +180,9 @@ description =
     # W605: invalid escape sequence ‘x’
     # See https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
 deps = pycodestyle
-commands = pycodestyle --select E111,E115,E21,E221,E222,E225,E227,E228,E25,E271,E272,E275,E302,E303,E305,E306,E401,E502,E701,E702,E703,E71,E72,W291,W293,W391,W605 {posargs:{toxinidir}/sage/}
-       pycodestyle --select E111,E222,E271,E301,E302,E303,E305,E306,E401,E502,E703,E712,E713,E714,E72,W29,W391,W605, --filename *.pyx {posargs:{toxinidir}/sage/}
+commands =
+    pycodestyle --select E111,E115,E21,E221,E222,E225,E227,E228,E25,E271,E272,E275,E302,E303,E305,E306,E401,E502,E701,E702,E703,E71,E72,W291,W293,W391,W605 {posargs:{toxinidir}/sage/}
```

Comment:
What do you think about completely removing everything related to pycodestyle from the code base? Or do you think its worth to maintain this still in parallel to ruff?

Reply:
this also checks cython files, so let us keep that for the moment

Reply:
Fair enough. For reference, cython support in ruff is https://github.com/astral-sh/ruff/issues/10250.